ci-operator/step-registry/cucushift/upgrade/mirror-images/by-oc-adm/cucushift-upgrade-mirror-images-by-oc-adm-ref.yaml

```diff
@@ -1,13 +1,10 @@
 ref:
   as: cucushift-upgrade-mirror-images-by-oc-adm
-  from_image:
-    namespace: ci
-    name: verification-tests
-    tag: latest
+  from: upi-installer
```

Comment:
since the issue of verification-tests has been fixed in another pr, wondering why we change the image here.

Reply:
When the permission issue of loading .config/containers/registries.conf.d was happening, I found `mirror-images-by-oc-adm` always succeeded, that make me a bit confused why the install one passed, but the upgrade one failed. After checking the difference between the two steps, the difference is about the image setting in the two steps. For `cucushift-upgrade-mirror-images-by-oc-adm` step, actually it does not rely on `verification-tests` image, so I changed it to keep consistent with `mirror-images-by-oc-adm`.

Reply:
ah, got it